libraries/AP_IOMCU/iofirmware/ioprotocol.h

```diff
@@ -184,6 +189,13 @@ struct __attribute__((packed, aligned(2))) page_GPIO {
     uint8_t output_mask;
 };
 
+struct __attribute__((packed, aligned(2))) page_mode_out {
```

Comment:
i think we should remove the packed, aligned attributes

Reply:
Fixed